app/views/npq_separation/admin/users/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= govuk_back_link(href: url_for(:back)) %>
```

Comment:
I think back buttons have not been added to the new admin, but happy to keep this for now